src/main/webapp/WEB-INF/views/drivers/all.jsp

```diff
@@ -5,7 +5,7 @@
 </style>
 <html>
 <head>
-    <title>All drivers</title>
+    <title>All logins</title>
```

Comment:
why all logins? I asked to display login for each driver (like you display their name and id, and license number)

Reply:
Ok. I was confused a litte bit)